src/client_side_request.cc

```diff
@@ -1576,8 +1576,6 @@ ClientHttpRequest::sslBumpEstablish(Comm::Flag errflag)
         return;
     }
 
-    // We lack HttpReply which logRequest() uses to log the status code.
-    // TODO: Use HttpReply instead of the "200 Connection established" string.
     al->http.code = 200;
```

Comment:
Should this assignment be removed since we do have an ALE response now?

Reply:
We can remove since ClientHttpRequest::logRequest() assigns it from al->reply.

Reply:
If this assignment is useless, let's remove it. Otherwise, let's add a comment to explain why it is here despite us supplying an HttpReply.